### ISSUE TITLE
Install core plugins via composer

### ISSUE THREAD

**Issue by mapkyca** [idno/known#2361]
**Is your feature request related to a problem? Please describe.**

It could be useful to move core plugins out of the central repo and to install them via composer.

This would make the framework download much lighter, and make it much easier for people to roll their own customisation builds.

**Describe the solution you'd like**

Plugins exist in their own repo and are installed via composer and composer installers

**Describe alternatives you've considered**

Basically only the above

**Additional context**
Depends on #2359 & #2353
